pages/events/index.vue

```diff
@@ -156,57 +193,81 @@ export default class EventsPage extends QiskitPage {
     }
   ]
 
-  get noEvents (): boolean {
-    return (this as any).filteredEvents.length === 0
-  }
+  calendars = [
+    {
+      name: 'Google',
+      component: GoogleCalendar
+    },
+    {
+      name: 'Outlook',
+      component: OutlookCalendar
+    },
+    {
+      name: 'Apple',
+      component: AppleCalendar
+    }
+  ]
 
-  getCheckedFilters (filter: string) {
-    return (this as any)[filter]
-  }
+    qiskitCalendarSyncLink = 'https://qisk.it/calendar-sync'
+    qiskitCalendarLink: GeneralLink = {
+      url: 'https://qisk.it/calendar',
+      label: 'https://qisk.it/calendar',
```

Comment:
~This label should be more helpful to the user. It could be something like "Calendar of Qiskit events".~

Actually, `label` is not part of the anchor elements' attributes and it doesn't have any effect to set it: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a?retiredLocale=de#attributes

We should be using the `title` attribute instead.

But this is pointing to a bigger issue that might be affecting multiple links in our page and that is out of the scope of this pull request. We might want to create an issue to address this.

Reply:
I'm only using `label` because that's what it's called in the `GeneralLink` [interface](https://github.com/Qiskit/qiskit.org/blob/main/constants/appLinks.ts#L8-L10)

Just pointing out that we're _not_ using it as an anchor tag attribute, but only to render the text of the anchor... 

Maybe the interface could be refactored? You're right though, it'll involve a lot of changes across the whole application.